lib/cext/preprocess.rb

```diff
@@ -30,7 +35,44 @@
 
       "#{shim}(#{argc}, #{argv}, \"#{arity}\", "
     }
+  elsif line =~ /^(\s+)VALUE((?:\s+\w+\s*(?:\[\s*\d+\s*\]\s*)?,?)+);\s*$/
```

Comment:
Done, but I did `/^(\s+)VALUE\s+((?:\s*\w+\s*(?:\[\s*\d+\s*\]\s*)?,?)+);\s*$/`, with the new `\s+` after `VALUE` because you need a space after that.